### PR TITLE
Featured sorter often requested

### PR DESCRIPTION
**Pull request by ajeh**

### CODE REVIEW

**Comment by DivaVocals** [comment]
:+1: Fan-freakin-tastic!!!

File: admin/featured.php [hunk 1]
```diff
@@ -12,9 +12,19 @@
...
+  }
+
+  zen_set_time_limit(900);
 
   if (zen_not_null($action)) {
     switch ($action) {
+      case 'set_featured_sort_order':
+        $_SESSION['featured_sort_order'] = $_GET['reset_featured_sort_order'];
+        $action='';
+        zen_redirect(zen_href_link(FILENAME_FEATURED, (isset($_GET['page']) && $_GET['page'] > 0 ? 'page=' . $_GET['page'] . '&' : '') . (isset($_GET['sID']) && $_GET['sID'] > 0 ? 'sID=' . $_GET['sID'] : '') . (isset($_GET['search']) && trim($_GET['search']) != ''  ? '&search=' . $_GET['search'] : '')));
```
Comment (drbyte) [inline admin/featured.php]: You're using `$_GET['sID']` here, but this file is for "Featured", and I think it should be `fID` instead.

Note the code from 6 lines further down:

```php
  zen_redirect(zen_href_link(FILENAME_FEATURED, (isset($_GET['page']) ? 'page=' . $_GET['page'] . '&' : '') . 'fID=' . $_GET['id'] . (isset($_GET['search']) ? '&search=' . $_GET['search'] : ''), 'NONSSL'));
```

**Comment by ajeh** [comment]
Thanks for the catch!

**Comment by drbyte** [comment]
:+1:

**Comment by drbyte** [comment]
:+1:

**Comment by zcwilt** [comment]
:+1:

**Pull request merged**